### PR TITLE
minify service worker script before html inlining (#59)

### PR DESCRIPTION
**Pull request by anubhav7495**
I have added the solution to minify the `service-worker-prod.js` before inlining in index.html.

The approach works for me in production here: [hnpwa-vue](https://anubhav7495.github.io/hnpwa-vue/#/news)

Requesting reviews.
Thanks.

### CODE REVIEW

**Comment by anubhav7495** [comment]
@addyosmani Can you please take a look at this?

**Comment by addyosmani** [comment]
@anubhav7495 Btw, just in case you might be interested in helping out with this repo too I've added you as a collaborator to it :)

File: template/build/webpack.prod.conf.js [hunk 1]
```diff
@@ -10,6 +10,7 @@ var HtmlWebpackPlugin = require('html-webpack-plugin')
 var ExtractTextPlugin = require('extract-text-webpack-plugin')
 var OptimizeCSSPlugin = require('optimize-css-assets-webpack-plugin')
 var SWPrecacheWebpackPlugin = require('sw-precache-webpack-plugin')
+var loadMinified = require('./load-minfied')
```
Comment (Gomah) [inline template/build/webpack.prod.conf.js]: @anubhav7495 Typo here?

Comment (anubhav7495) [inline template/build/webpack.prod.conf.js]: Thanks. I will push a fix immediately.

**Comment by anubhav7495** [comment]
@addyosmani Thanks.
@Gomah PR for Typo Fix: https://github.com/vuejs-templates/pwa/pull/67

**Comment by addyosmani** [comment]
Cheers, @anubhav7495!

**Pull request merged**